moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -410,6 +530,14 @@ class OnlineRobot : public Robot
 
   // ROS objects
   ros::Subscriber j_state_sub_;
+
+  // Dynamic Reconfigure
+  double speed_modifier_;
```

Comment:
I know this is based on my original code, but the terminology and sematics of this are bad.  Can we change it to be `speed_percent` with values from 0->1.0?

Reply:
ok, sure, I can do that.